examples/api-routes-apollo-server-and-client/lib/with-apollo-client.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import initApollo from './init-apollo'
+import Head from 'next/head'
+import { getDataFromTree } from '@apollo/react-ssr'
+import { ApolloProvider } from '@apollo/react-hooks'
+
+export default App => {
+  return class Apollo extends React.Component {
+    static displayName = 'withApollo(App)'
```

Comment:
Lets use the real display name otherwise it will be misleading.

e.g.: https://github.com/zeit/next.js/blob/canary/examples/with-apollo/lib/apollo.js#L35